src/components/pages/MatchUp/MatchUpContainer.js

```diff
@@ -76,12 +62,10 @@ function MatchUpContainer({ LoadingComponent, ...props }) {
 export default connect(
   state => ({
     child: state.child,
-    faceoffs: state.faceoffs,
```

Comment:
like here is there a specific reason we deleted these redux elements, I know that faceoffs and squads are the same. Team E just named it squad and our team named it faceoffs.

Reply:
yes just difference in naming I guess..  I can change the name back to  faceoffs..